Assets/MixedRealityToolkit.Extensions/LostTrackingService/Scripts/BasicLostTrackingVisual.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Tracking
+{
+    /// <summary>
+    /// A basic lost tracking visual for HoloLens devices.
+    /// </summary>
+    public class BasicLostTrackingVisual : MonoBehaviour, ILostTrackingVisual
+    {
+        [SerializeField]
+        private MeshRenderer gridRenderer = null;
+        [SerializeField]
+        private AudioClip loopClip = null;
+        [SerializeField]
+        private AudioSource audioSource = null;
+        [SerializeField]
+        private float pulseTimer = 0.0f;
+        [SerializeField]
+        private float pulseDuration = 2.0f;
+
+        public bool Enabled
+        {
+            get { return gameObject.activeSelf; }
+            set { gameObject.SetActive(value); }
+        }
+
+        public void ResetVisual()
```

Comment:
This is missing comments. One of our principles is "comment all public methods" so they show up in docfx.

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6408